notebooks/213-question-answering/213-bert-question-answering.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        result = model.infer(inputs=inputs)
</pre><p>Do you mean <code></code><code>complied_model</code><code></code> here ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Yes, I have just pushed the correct one